src/nfa/nfa.cc

```diff
@@ -148,28 +146,28 @@ namespace {
     /**
      * Get a new trimmed automaton.
      * @param[in] nfa NFA to trim.
-     * @param[in] original_to_new_states_map Map of old states to new trimmed automaton states (new states should follow the ordering of old states).
+     * @param[in] original_to_new_state_renaming Map of old states to new trimmed automaton states (new states should follow the ordering of old states).
      * @return Newly created trimmed automaton.
      */
-    Nfa create_trimmed_aut(const Nfa& nfa, const StateToStateMap& original_to_new_states_map) {
-        Nfa trimmed_aut{ original_to_new_states_map.size() };
+    Nfa create_trimmed_aut(const Nfa& nfa, const StateRenaming& original_to_new_state_renaming) {
```

Comment:
```suggestion
    Nfa create_trimmed_aut(const Nfa& nfa, const StateRenaming& state_renaming) {
```
I personally think that `original_to_new` is given from the usage of the parameter. Or if you want to convey it is out parameter, then add `out_state_renaming`. The current is overkill.

Reply:
I will rename the variable.